src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1

```diff
@@ -219,7 +234,7 @@ param(
   Set-PowerShellExitCode $exitCode
   Write-Debug "Command ['$7zip' $params] exited with `'$exitCode`'."
 
-  if ($zipExtractLogFullPath) {
+  if ($zipExtractLogFullPath -and !$disableLogging) {

```

Comment:
@vexx32 am I right in saying that this is not the preferred way to do a `!` in PowerShell?

Reply:
in general I would tend to prefer `-not`, yes. Either works in practice, though.

Reply:
Done.